assets/_scss/elements/_inputs.scss

```diff
@@ -147,19 +141,19 @@ input[type="checkbox"] + label::before,
 input[type="radio"] + label::before {
   background: white;
   border-radius: $border-radius;
-  box-shadow: 0 0 0 1px #757575;
+  box-shadow: 0 0 0 1px $color-gray-medium;
   content: '\a0';
   display: inline-block;
   height: 1.8rem;
-  line-height: .8;
-  margin-right: .6em;
-  text-indent: .15em;
-  vertical-align: .2em;
+  line-height: 0.8;
+  margin-right: 0.6em;
```

Comment:
There's a mix of `em` vs `rem` units in this file and I'm not sure if its' intentional.